carbon-design-system/carbon
# feat(motion): add duration tokens to motion

Closes #7738 

Adds support for duration tokens to the motion package.

#### Changelog

**New**

- Add tests for public API of motion package

**Changed**

- Update module style and JS entrypoints to include duration tokens

**Removed**

#### Testing / Reviewing

- Verify token values are correct
- Verify tokens appear correctly in snapshots for the motion package
